src/main.rs

```diff
@@ -0,0 +1,116 @@
+//! Actix Basic
+//!
+//! Tasks defined `rust_actix-basic.pdf`
+//!
+//! - Modeling endpoint behavior base on [this test api](https://jsonplaceholder.typicode.com/api/v1/users)
+//! - This server implementation based on [this basic example](https://github.com/actix/examples/blob/master/basics/src/main.rs)
+
+use ::serde_json;
+use simplelog::*;
+use std::{env, io};
+use validator::Validate;
+
+#[macro_use]
+extern crate actix_web;
+use actix_files as fs;
+use actix_web::http::StatusCode;
+use actix_web::{error, middleware, web, App, Error, HttpResponse, HttpServer, Result};
+
+mod user;
+
+/// favicon handler
+#[get("/favicon.ico")]
+async fn favicon() -> Result<fs::NamedFile> {
+    Ok(fs::NamedFile::open("static/favicon.ico")?)
+}
+
+/// simple index handler
+#[get("/health")]
+async fn health() -> Result<HttpResponse> {
+    Ok(HttpResponse::build(StatusCode::OK)
+        .content_type("text/plain; charset=utf-8")
+        .body("ok"))
+}
+
+/// 404 handler
+async fn p404() -> Result<HttpResponse> {
+    Ok(HttpResponse::new(StatusCode::NOT_FOUND))
+}
+
+/// User GET requests specifying the API version `/api/{version}/users`
+async fn get_users(vers: web::Path<(String,)>) -> Result<HttpResponse> {
+    vers_check(&vers)?;
+
+    Ok(HttpResponse::build(StatusCode::OK)
+        .content_type("application/json; charset=utf-8")
+        .body(include_str!("../static/users.json")))
```

Comment:
Call the actual api and return the values now.  After you do that, you can delete the users.json file.